templates/solutions/ai/index.html

```diff
@@ -67,226 +71,225 @@ <h2>
             }}
           </div>
-          <div class="p-logo-section__item u-no--margin-right">
-            {{ image(url="https://assets.ubuntu.com/v1/4678094b-oracle-cloud-logo.png",
+        </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Big data analytics</h3>
+        </div>
+      </a>
+
+      <a class="p-equal-height-row__col"
+         href="/solutions/ai/edge"
+         aria-label="Edge AI">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container--16-9">
+            {{ image(url="https://assets.ubuntu.com/v1/dd383834-edge-ai.png",
                         alt="",
-                        width="478",
-                        height="384",
+                        width="852",
+                        height="481",
                         hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
+                        loading="auto",
+                        attrs={"class": "p-image-container__image"}) | safe
             }}
           </div>
         </div>
-      </div>
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Edge AI</h3>
+        </div>
+      </a>
     </div>
   </section>
 
   <section class="p-section">
     <div class="row--50-50">
       <hr class="p-rule" />
       <div class="col">
-        <h2>Scale with MLOps</h2>
+        <h2>
+          Kick-start
+          <br />
+          your enterprise AI journey
+        </h2>
       </div>
       <div class="col">
         <p>
-          Simplify workflow processes and automate machine learning deployments at any scale with our modular MLOps solutions. Our secure and supported Kubeflow distribution, integrated with a growing ecosystem of AI and data tooling, brings scalability, portability and reproducibility to your machine learning operations.
+          The first step is always the hardest, especially with AI expertise in such short supply. Canonical’s professional services will help you bridge the skills gap and overcome the barriers to AI adoption.
         </p>
-        <hr class="p-rule" />
         <p>
-          <a href="https://ubuntu.com/engage/mlops-guide">Read our guide to MLOps&nbsp;&rsaquo;</a>
+          Upskill your team and lay the groundwork for your projects with a 5-day MLOps workshop, work with us to build a PoC, or completely offload the AI operational burden with managed services.
         </p>
-        {{ image(url="https://assets.ubuntu.com/v1/d5596eaf-Image-2.png",
-                alt="",
-                width="1200",
-                height="676",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
+        <hr class="p-rule--muted" />
+        <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Datasheet%20for%20MLOps%20workshop.pdf?version=0">Download the AI workshop datasheet&nbsp;&rsaquo;</a>
```

Comment:
Please use `p-cta-block` here and at other places that have a call to action.
Refer here https://vanillaframework.io/docs/patterns/cta-block